### ISSUE TITLE
Increased 'WARN' rate due to 'Single block lookup failed'

### ISSUE THREAD

**Issue by gotzl** [sigp/lighthouse#2446]
## Description
My two lighthouse beacon nodes frequently report a warning in the logs:
```
Jul 10 23:21:40.821 WARN Single block lookup failed              outcome: Err(WouldRevertFinalizedSlot { block_slot: Slot(1594432), finalized_slot: Slot(1594528) }), service: sync
```
I am not sure what it means and what I can/should do to get rid of it.

## Version
v1.4.0-3b600ac

## Present Behaviour
I run two lighthouse beacon nodes on two separate devices. I monitor the warn rate of these instances, which is usually ~1/h. However, since approx 10.07., both instances report a significantly higher warn rate (~20/h). Most (or all) of the warnings are messages as quoted above.
I cannot classify this warning nor do I know how to counteract. A google search didn't help, so I would like to ask here for enlightenment.

The nodes are on mainnet, I use `openethereum` for eth1.

## Expected Behaviour
No warnings should be logged.

## Steps to resolve
Unknown.

**Comment by AgeManning** [comment]
Thanks for raising this.

Rest assured this warning is no fault of your lighthouse nodes. We are also seeing this on our lh nodes.

The issue appears to be a collection of prysm nodes (of the ones I checked, were running this version: `Prysm/v1.3.10/5ae6e83c7f6238593c6dfa3a8e05fc7c3778783f`) that are not validating correctly on the network. It appears they have not synced their time correctly and are referencing old blocks in attestations. 

We have this behaviour as a warning in lighthouse, mainly to help us track malicious behaviour on the networks. The warn's stand out. Your lighthouse node will recognise this bad behaviour and kick the bad peer, however it seems there are a few bad peers so they seem to recycle and reconnect and reproduce the same behaviour. 

Although Lighthouse does ban these nodes, our ban's eventually get lifted. We will likely increase the length of the ban to prevent nodes like this from eventually reconnecting (and hence reduce/eliminate the warnings you are noticing)

**Comment by gotzl** [comment]
Thanks for confirmation and the detailed explanation, and the 'fix' ;)

**Comment by AgeManning** [comment]
I'm going to mark this as closed. The ban length increase should reduce the occurrence. The operators of the bad nodes should be losing eth and I imagine they will discover and remedy their nodes soon. If these warnings persist we can downgrade the log to a debug. 
The ban-timeout will be released in the upcoming `1.5` release.
